.github/workflows/main.yml

```diff
@@ -22,4 +22,4 @@ jobs:
         env:
           packageUser: ${{ github.actor }}
           packagePAT: ${{ secrets.GITHUB_TOKEN }}
-        run: ./gradlew clean build --stacktrace --scan --console=plain --no-daemon
+        run: ./gradlew clean build -x test --stacktrace --scan --console=plain --no-daemon
```

Comment:
Why do we exclude the tests?

Reply:
Please check https://github.com/ballerina-platform/ballerina-distribution/issues/628. Once we fixed that, we can enable this back.